docs/config-file-package-configuration-yml.md

```diff
@@ -51,24 +56,69 @@ curations have the same syntax and semantic as the analog entries for the `ort.y
     concluded_license: "Apache-2.0"
 ```
 
-## Organizing package configurations
+## Command Line
 
-ORT currently provides two alternatives for organizing package configurations to choose from. It is required for both
-options that there is at most one package configuration provided for any (package, provenance) combination.
+ORT offers two different ways to use package configurations:
 
-### Configuration directory
+- A single configuration `.yml` containing an array with each entry defining the configuration for one package
+- A directory with configuration files for each `package id`, with one file for each configured package/provenance combination
```

Comment:
each `package id` VS. each configured package/provenance combination

..is contradictory. the latter is the right one.

Reply:
Fixed